DESCRIPTION

```diff
@@ -37,4 +37,4 @@ Suggests:
     knitr,
     rmarkdown
 Enhances: magrittr
-RoxygenNote: 6.0.1
+RoxygenNote: 6.0.1.9000
```

Comment:
I can take out the .9000 at the end -- just have been removing it every time and figured it makes sense to only do it once, at the very end :)